common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -389,16 +389,8 @@ div.problem {
       }
     }
 
-    .unanswered {
-      p.status.drag-and-drop--status {
-        @include margin(8px, 0, 0, ($baseline/2));
-        text-indent: 100%;
-        white-space: nowrap;
-        overflow: hidden;
-      }
-    }
-
     &.correct, &.ui-icon-check {
+      // JSME and possibly other "old" capa problem types still rely on this class structure.
       p.status {
```

Comment:
Actually, it appears that the status indicator isn't working for JSME in production anyway. So it looks like I might as well delete these classes and then perhaps try to fix JSME.